added initial karma config & test for browser testing (#306)
lbdremy/solr-node-client: README.md

npm run solr:legacy:start
npm run test:legacy
```
## Windows 10 and Docker

#### Windows 10 Pro or Enterprise

If you have a version of Windows 10 Pro or Enterprise, simply download [Docker Desktop](https://docs.docker.com/desktop/windows/install/). Ensure that Hyper-V is enabled, and [virtualization](https://docs.docker.com/desktop/windows/troubleshoot/#virtualization-must-be-enabled) is enabled in your motherboard's BIOS. It may prompt you to install updates to Windows during Docker Desktop installation and then restart the system.

#### Windows 10 Home

> If you have a version of Windows 10 Home, a little more effort is needed. By default, Hyper-V is not able to be turned on for Home.

Reviewer: ```suggestion
If you have a version of Windows 10 Home, a little more effort is needed. By default, Hyper-V cannot be turned on for Home edition.
```

> **The steps I found to work:**

Reviewer: ```suggestion
**The steps that seem to work:**
```
